src/benchmark/metrics.py

```
@@ -0,0 +1,488 @@
+    Semantic similarity using cosine similarity in embedding space.
+
+    Embeds both predicted answer and ground truth answers using a semantic
+    similarity model (all-MiniLM-L6-v2), then computes cosine similarity.
+
+    This measures semantic similarity rather than structural/lexical similarity:
+    - Paraphrases score high (semantic match)
+    - Word order doesn't matter (meaning-based)
+    - Captures conceptual similarity
+
+    Uses sentence-transformers with cosine similarity:
+    cos_sim = (A · B) / (||A|| × ||B||)
```

Ruff (0.14.2)

Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)